### PR TITLE
Fix broken installation of CrackMapExec

### LINKED ISSUE #124: Installation of CrackMapExec failing
# 🐛 Summary #

The installation of the [`CrackMapExec`](https://github.com/byt3bl33d3r/CrackMapExec) tool [is currently failing](https://github.com/cisagov/kali-packer/actions/runs/3421627136/jobs/5698067418).

It appears that the `aardwolf` dependency of `CrackMapExec` is [only available as a source download](https://pypi.org/project/aardwolf/#files) (i.e., there is no wheel available for download).

## To reproduce ##

1. Attempt to build an AMI from this repository.
1. Watch the build process fail as it does [here](https://github.com/cisagov/kali-packer/actions/runs/3421627136/jobs/5698067418).

## Expected behavior ##

The AMI should build cleanly.

## Any helpful log output or screenshots ##

See [here](https://github.com/cisagov/kali-packer/actions/runs/3421627136/jobs/5698067418).

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request:
- Pulls in the changes from cisagov/ansible-role-assessment-tool#42
- Leverages the aforementioned changes to fix a bug that is currently causing AMI builds to fail when installing [`CrackMapExec`](https://github.com/Porchetta-Industries/CrackMapExec)
- Modifies the URL from which `CrackMapExec` is installed

Note that we will build new staging and prod COOL AMIs after pulling these changes into #126.

## 💭 Motivation and context ##

- Resolves #124.
- The old URL from which `CrackMapExec` was being installed now redirects to the new one, so we may as well get the code from the true source.

## 🧪 Testing ##

All automated tests pass.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [x] Revert to using the default branch of [cisagov/ansible-role-assessment-tool](https://github.com/cisagov/ansible-role-assessment-tool) once cisagov/ansible-role-assessment-tool#42 is merged.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
Excellent 👍

File: src/crackmapexec.yml [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
+# * gssapi requires the krb5-config executable from the libkrb5-dev
+#   package
+# * aardwolf requires a Rust compiler, as the Python package is only
+#   available from PyPI.org as a source distribution (i.e., there is
+#   no wheel available for download)
 - name: Install CrackMapExec dependencies
   ansible.builtin.package:
     name:
       - libkrb5-dev
+      - rustc
```
Comment (mcdonnnj) [inline src/crackmapexec.yml]: Should we just install `cargo` since that's Rust's preferred method for environment set up?

**Review by mcdonnnj — APPROVED** [review]
Makes sense. I have one question for your consideration.

File: src/crackmapexec.yml (hunk 1 above)
Comment (jsf9k) [reply src/crackmapexec.yml]: I'm really only interested in adding enough stuff for Python to build the wheel for `aardwolf`.  I could add `cargo`, but it doesn't seem to be necessary to build the wheel.

If the assessors ever requested us to add a Rust tool to this AMI, then I would modify [cisagov/ansible-role-assessment-tool](https://github.com/cisagov/ansible-role-assessment-tool) to support building a Rust project.  That would presumably involve installing `cargo`, and I would reuse that functionality here.  Do you want me to add an issue to that repo to support Rust (and reference this PR)?  Would that be enough to scratch your `cargo` itch?

Comment (mcdonnnj) [reply src/crackmapexec.yml]: That will work for me. I ran intro troubles where I needed to install `cargo` for a Python package to successfully build its Rust extensions so I just wanted to mention it as an option.

Comment (jsf9k) [reply src/crackmapexec.yml]: @mcdonnnj, please see:
- cisagov/ansible-role-assessment-tool#41
- cisagov/ansible-role-assessment-tool#42
- fb16595
- 464f5d8

**Pull request merged**